compiler/testData/codegen/box/reflection/callBy/valueClasses/nonNullObject/defaultArguments/constructorWithInlineClassParameters.kt

```diff
@@ -0,0 +1,124 @@
+// TARGET_BACKEND: JVM
+// WITH_REFLECT
+
+import kotlin.test.assertEquals
+
+inline class A(val x: String)
+
+data class TestCtor1_1(val x: A = A("0"))
+data class TestCtor1_2(val x: A? = A("0"))
```

Comment:
I am adding a function to verify the `nullability` of a parameter.
Should I add a case where the default argument is `null`?